cmk.py

```diff
@@ -79,6 +81,8 @@
   --namespace=<name>           Set the namespace to deploy pods to during the
                                cluster-init deployment process.
                                [default: default].
+  --excl-non-isolcpus=<list>   List of cores to be added to the extra exclusive
```

Comment:
nit: it's a list of *physical* cores - passing logical cores that exist will throw an error anyway